src/Platform/Microsoft.Testing.Platform/Hosts/TestHostBuilder.cs

```diff
@@ -255,7 +255,7 @@ public async Task<ITestHost> BuildAsync(
         {
             string arg = args[0];
             int size = arg.Length;
-            if (!float.TryParse(arg[..(size - 1)], out float value))
+            if (!float.TryParse(arg[..(size - 1)], CultureInfo.InvariantCulture, out float value))
```

Comment:
src/Platform/Microsoft.Testing.Platform/Hosts/TestHostBuilder.cs(258,24): error CS7036: (NETCORE_ENGINEERING_TELEMETRY=Build) There is no argument given that corresponds to the required parameter 'result' of 'float.TryParse(string, NumberStyles, IFormatProvider, out float)'